testscommon/factory/bootstrapperFactoryMock.go

```diff
@@ -0,0 +1,24 @@
+package factory
+
+import (
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/process/sync"
+)
+
+// BootstrapperFactoryMock -
+type BootstrapperFactoryMock struct {
+	CreateBootstrapperCalled func(argsBaseBootstrapper sync.ArgShardBootstrapper) (process.Bootstrapper, error)
+}
+
+// CreateBootstrapper -
+func (b *BootstrapperFactoryMock) CreateBootstrapper(argsBaseBootstrapper sync.ArgShardBootstrapper) (process.Bootstrapper, error) {
+	if b.CreateBootstrapperCalled != nil {
+		return b.CreateBootstrapperCalled(argsBaseBootstrapper)
```

Comment:
Same comment regarding naming these stubs + return a stub/mock in case the func is nil. 
! Applies in all files

Reply:
added return a stub/mock for components that had a mock/stub in testscommon